src/schema/users.ts

```diff
@@ -238,6 +239,11 @@ export const typeDefs = /* GraphQL */ `
       query: String!
     ): SearchReadingHistorySuggestionsResults!
 
+    """
+    Get user's info
+    """
+    userInfo(id: ID!): User
```

Comment:
Why do we need this new query?

Reply:
This is so we can do a batch request on our tooltip data requirements instead of making two subsequent requests.